### PR TITLE
Add EMR VC tag in props

### PR DESCRIPTION
**Pull request by lmouhib**
*Issue #, if available:*

*Description of changes:*
Add the ability to add `tags` in EMR on EKS team props. These tags are applied to EMR on EKS virtual cluster.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: lib/teams/emr-on-eks-team.ts [hunk 1]
```diff
@@ -47,9 +47,13 @@ export interface EmrEksTeamProps extends TeamProps {
   */
   virtualClusterName: string,
   /**
-   * List of execution role to associated with the VC namespace
+   * List of execution role to associated with the VC namespace {@link ExecutionRoleDefinition}
    */
-  executionRoles: ExecutionRoleDefinition[] 
+  executionRoles: ExecutionRoleDefinition[]
```
Comment (shapirov103) [inline lib/teams/emr-on-eks-team.ts:52]: minor: semicolon after each member of class and interface. Please refer to these code conventions: https://google.github.io/styleguide/tsguide.html#member-property-declarations

File: lib/teams/emr-on-eks-team.ts [hunk 2]
```diff
@@ -58,7 +62,7 @@ export interface EmrEksTeamProps extends TeamProps {
  *It can either create a namespace or use an existing one
  *The class will set the necessary k8s RBAC needed by EMR on EKS as defined in the AWS documentation 
  * https://docs.aws.amazon.com/emr/latest/EMR-on-EKS-DevelopmentGuide/setting-up-cluster-access.html
- * The class constructor take a the EMR on EKS Team definition
+ * The class constructor take an EMR on EKS Team definition and need the EMR on EKS AddOn added
```
Comment (shapirov103) [inline lib/teams/emr-on-eks-team.ts:65]: class constructor takes EMR on EKS team definition as a parameter. Pre:requisite: EMS on EKS AddOn is part of the blueprint.

**Review by shapirov103 — COMMENTED** [review]
minor comments, I will kick off test.

File: lib/teams/emr-on-eks-team.ts [hunk 3]
```diff
@@ -47,9 +47,13 @@ export interface EmrEksTeamProps extends TeamProps {
   */
   virtualClusterName: string,
   /**
-   * List of execution role to associated with the VC namespace
+   * List of execution role to associated with the VC namespace {@link ExecutionRoleDefinition}
    */
-  executionRoles: ExecutionRoleDefinition[] 
+  executionRoles: ExecutionRoleDefinition[],
```
Comment (shapirov103) [inline lib/teams/emr-on-eks-team.ts:52]: As I mentioned in my prev comment: It should be a semicolon `;` as opposed to comma and applied consistently across all interface members.

File: lib/teams/emr-on-eks-team.ts [hunk 4]
```diff
@@ -58,8 +62,8 @@ export interface EmrEksTeamProps extends TeamProps {
  *It can either create a namespace or use an existing one
  *The class will set the necessary k8s RBAC needed by EMR on EKS as defined in the AWS documentation 
  * https://docs.aws.amazon.com/emr/latest/EMR-on-EKS-DevelopmentGuide/setting-up-cluster-access.html
- * The class constructor take a the EMR on EKS Team definition
- * The EmrEksTeam will `throw` an error if the EMR on EKS AddOn is not added
+ * class constructor takes EMR on EKS team definition as a parameter. Pre:requisite: EMS on EKS AddOn is part of the blueprint.
```
Comment (shapirov103) [inline lib/teams/emr-on-eks-team.ts:65]: EMS on EKS AddOn? Don't you mean EMR (it was correct before).

**Review by shapirov103 — COMMENTED** [review]
@lmouhib A couple of minor lingering comments

File: lib/teams/emr-on-eks-team.ts (hunk 3 above)
Comment (lmouhib) [reply lib/teams/emr-on-eks-team.ts]: fixed

File: lib/teams/emr-on-eks-team.ts (hunk 4 above)
Comment (shapirov103) [reply lib/teams/emr-on-eks-team.ts]: It was a random attention test ;)

**Review by shapirov103 — COMMENTED** [review]
LGTM

**Review by shapirov103 — APPROVED** [review]
LGTM

**Pull request merged**